LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -2577,6 +2579,52 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
                         c.Add(new Control("Chassis Fan #3", 6));
                         break;
 
+                    case Model.Z790_Nova_WiFi:
```

Comment:
Can you please follow the other names for the sensors? Things like T_SENSOR1 should be `Temperature #3` for example.

Reply:
Only joining in here since I should mention that `T_Sensor#` temperature sensors on this board are specifically for external sensors. Probably would be best to reflect that instead of a generic `Temperature #`.

Reply:
@PhyxionNL  maybe it can rename to `Temperature #3 (T_SENSOR1)`?

Reply:
External Temperature #x then?

Reply:
Like `T_SENSOR1` to `Externel Temperature #1`?